rcp/org.testeditor.rcp4.views.tcltestrun/src/org/testeditor/rcp4/views/tcltestrun/model/TestExecutionManager.xtend

```diff
@@ -94,7 +94,11 @@ class TestExecutionManager {
 
 	def TestRunStatistic readTestStatistic(File parentDir) {
 		val docBuilder = DocumentBuilderFactory.newInstance().newDocumentBuilder
```

Comment:
val docBuilder = DocumentBuilderFactory.newInstance.newDocumentBuilder; // no braces
this codeline could as well be moved down to after the if, right ?